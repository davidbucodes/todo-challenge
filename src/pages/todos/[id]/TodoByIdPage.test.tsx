import { composeStories } from "@storybook/react";
import { render, screen } from "@testing-library/react";
import * as stories from "./TodoByIdPage.stories";

const { Default } = composeStories(stories);

describe("TodoByIdPage", () => {
  describe("Default", () => {
    it("should render correctly", () => {
      const { container } = render(<Default />);
      const wrapperElement = screen.getByText(/TodoByIdPage/i);
      expect(container).toMatchSnapshot();
      expect(wrapperElement).toBeInTheDocument();
    });
  });
});
