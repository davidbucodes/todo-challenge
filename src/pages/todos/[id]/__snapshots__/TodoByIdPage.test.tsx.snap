// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoByIdPage Default should render correctly 1`] = `
<div>
  <div
    class="sc-bgqQcB"
  >
    TodoByIdPage
    <div
      class="sc-bgqQcB"
    >
      12345
    </div>
  </div>
</div>
`;
